pkg/apis/workloads/v1alpha1/console_types.go

```diff
@@ -20,9 +20,19 @@ type Console struct {
 
 // ConsoleSpec defines the specification for a console
 type ConsoleSpec struct {
-	User               string                      `json:"user"`
-	Reason             string                      `json:"reason"`
-	TimeoutSeconds     int                         `json:"timeoutSeconds"`
+	User   string `json:"user"`
+	Reason string `json:"reason"`
+
+	// Number of seconds that the console should run for.
+	// If the process running within the console has not exited before this
+	// timeout is reached, then the console will be terminated.
+	// Creation of a Console will be rejected if the specified timeout exceeds
+	// the maximum timeout defined within the ConsoleTemplate that the console is
+	// referring to.
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=604800
```

Comment:
Worth adding a comment explaining that this is 7 days?

Reply:
Originally didn't want to have this added to the API docs as it seemed redundant, but fair point.

Added